naughty/fedora-36/4031-pmie-check-cpu-usage

```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "/work/bots/make-checkout-workdir/test/verify/check-system-info", line *, in testOverview
+    wait(lambda: progressValue(1) < 20)
```

Comment:
This is bit too generic as there might be a lot of reasons why the load does not come under 20.  I would prefer if we would just disable `pmie.service`. Or is there some reason why we cannot do that?

Reply:
right, I was thinking about editing of the test. Good point about disabling pcp service in metrics test but this naughty does not say anything about what consumes the processor. If there is another service in the meantime we will still be blocking this one naughty and the test might never run to completion.

Reply:
So instead of checking the overall CPU, check the table of processes?

Reply:
Ok, created the workaround here https://github.com/cockpit-project/cockpit/pull/17892